mkdocs.yml

```diff
@@ -5,3 +5,32 @@ theme:
   name: readthedocs
 plugins:
   - search
+nav:
+    - Home:
+      - Integreatly Operator: index.md
+      - Getting Started: getting_started.md
+      - Terminology: terminology.md
+    - Installation: 
+      - Installation guides: installation_guides/installation_general_info.md
+      - Local installation: installation_guides/local_installation.md
+      - OLM Installation: installation_guides/olm_installation.md
+      - Identity Provider Setup: installation_guides/identity_provider_setup.md
+      - Additional configuration: installation_guides/additional_configuration.md
+      - Validation of installation: installation_guides/validation_of_installation.md
+    - Products:
+      - Products guides: products/products_general_info.md
+      - Adding new product: products/adding_new_product.md
+      - Configuring Custom Domain: products/custom_domain.md
+      - 3scale guides: products/3scale_guides.md
+    - Tests:
+      - Unit testing: tests/unit_tests.md
+      - End to End (e2e): tests/e2e.md
+      - Product tests: tests/product_tests.md
+      - Scorecard tests: tests/scorecard_tests.md
+    - Uninstallation:
+      - Uninstallation guides: uninstallation/uninstallation_general_info.md
+      - Local and OLM: uninstallation/local_and_olm.md
+      - Addon flow: uninstallation/addon_flow.md
+      - ROSA: uninstallation/rosa.md
```

Comment:
There is a couple points. 
Firstly with the change as it is there is a number of page where the page title does not match the nav item name. From what I have seen there is no reason why it needs to be done in this fashion.

Secondly why add this at all? Does the order matter? The name of the folders act as the section headers. If the section headers need to be updated changing the folder names would also work.  If possible I would like to avoid adding the nav section here. With out it any doc added to the docs folder is auto added to the site. It is less of maintenance overhead. If it is for ordering I understand that too.

Recap.
Some document titles do not match their navigation titles. This needs updating.
Why add the nav section? Is it to force the order sections are listed?

Reply:
Good point on naming, going to update.
As for nav, yes, without nav the order is alphabetical while to me docs should have a clear structure, for example, I do want to cover stuff like installation or pre-requisites before tests or uninstallation. We need nav configured IMO

Reply:
Actually, looking at it now, which names mismatch in your opinion?

Reply:
ah, nvm, I read your initial comment as the name of the files are different to the nav items...
going to update now, thanks

Reply:
@Boomatang pushed changes now, thanks